### DISCUSSION TITLE
Keybinds for color change

### DISCUSSION THREAD

**Discussion by fmorroni** [mbrlabs/Lorien#280] [Ideas]
Hi, it would be useful to be be able to change colors with keybindings. I'm thinking either adding something like a `change color` action that the user can bind to whatever he wants and pass the color as argument, or having just a primary and secondary color which you can switch between with a keybind.
The first option is more versatile but probably harder to implement. I think the second one would still be really useful.

**Comment by tafode** [comment]
Hi @fmorroni ,

that sounds similar, to what was requested in the isssue #251. A shortcut for the first 10 colors was requested there. And a pull request was already opened #273. Would that solve your problem?

**Reply by fmorroni** [reply]
Ah yes that would be perfect, not sure how I missed that issue. Thanks!
